bump_and_tag.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# This script is used for
+# a) creating a new tagged release of go.opentelemetry.io/contrib
+# b) bumping the referenced version of go.opentelemetry.io/otel
+#
+# The options can be used together or individually.
+#
+set -e
+
+help()
+{
+   printf "\n"
+   printf "Usage: %s [-o otel_tag] [-t tag]\n" "$0"
+   printf "\t-o Otel release tag. Update all go.mod to reference go.opentelemetry.io/otel <otel_tag>.\n"
+   printf "\t-t New unreleased tag. Update all go.mod with this tag.\n"
+   exit 1 # Exit script after printing help
+}
+
+while getopts "t:o:" opt
+do
+   case "$opt" in
+      t ) TAG="$OPTARG" ;;
+      o ) OTEL_TAG="$OPTARG" ;;
+      ? ) help ;; # Print help
+   esac
+done
+
+
+validate_tag() {
+    local tag_=$1
+    SEMVER_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(\\-[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?(\\+[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?$"
+    if [[ "${tag_}" =~ ${SEMVER_REGEX} ]]; then
+	    printf "%s is valid semver tag.\n" "${tag_}"
+    else
+	    printf "%s is not a valid semver tag.\n" "${tag_}"
+	    return 1
+    fi
+}
+
+# Print help in case parameters are empty
+if [[ -z "$TAG" && -z "$OTEL_TAG" ]]
+then
+    printf "At least one of '-o' or '-t' must be specified.\n"
+    help
+fi
+
+
+## Validate tags first
+if [ -n "${OTEL_TAG}" ]; then
+    validate_tag "${OTEL_TAG}" || exit $?
+    # check that OTEL_TAG is a currently released tag for go.opentelemetry.io/otel
+    TMPDIR=$(mktemp -d "/tmp/otel-contrib.XXXXXX") || exit 1
+    trap "rm -fr ${TMPDIR}" EXIT
+    (cd "${TMPDIR}" && go mod init tagtest)
+    # requires go 1.14 for support of '-modfile'
+    if ! go get -modfile="${TMPDIR}/go.mod" -d -v "go.opentelemetry.io/otel@${OTEL_TAG}"; then
+        printf "Otel tag %s does not appear to exist.\n" "${OTEL_TAG}"
```

Comment:
make clear in error msg that this is for `go.opentelemetry.io/otel` rather than for the -contrib repo.

Reply:
I changed the message to `go.opentelemetry.io/otel v1.0.0 does not exist. Please supply a valid tag`